pureskillgg_dsdk/tome/manifest.py

```diff
@@ -34,17 +42,30 @@ def end_page(self, page_number):
             "number": page_number,
             "keyset": {
                 "keyset": content_name(page_number, "keyset"),
-                "key": "/".join([self._tome_name, content_name(page_number, "keyset")]),
+                "key": "/".join(
+                    [
+                        self._ds_type,
+                        self._tome_name,
+                        "header" if self._is_copied_header else "",
+                        content_name(page_number, "keyset"),
+                    ]
+                ),
+                "ContentType": "application/x-parquet",
             },
             "dataframe": {
                 "dataframe": content_name(page_number, "dataframe"),
                 "key": "/".join(
-                    [self._tome_name, content_name(page_number, "dataframe")]
+                    [
+                        self._ds_type,
+                        self._tome_name,
+                        "header" if self._is_copied_header else "",
+                        content_name(page_number, "dataframe"),
+                    ]
                 ),
+                "ContentType": "application/x-parquet",
             },
-            "keysetContentType": "application/x-parquet",
-            "dataframeContentType": "application/x-parquet",
-            "createdAt": now_to_iso(),
+            "createdAt": self._current_page_start_time.isoformat(),
+            "completedAt": now_to_iso(),
```

Comment:
Ok I changed my mind on this. Just leave it at `createdAt: now_to_iso()` and kill `completedAt` for pages.

Reply:
kk